platform/posix/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Include filepaths for source and include.
+# include( "../platformFilePaths.cmake" )
+
+# PLATFORM library target.
+# add_library( platform
+#             ${PLATFORM_SOURCES} )
```

Comment:
Please remove file if not used

Reply:
Hmm.. This file is most likely not needed.  I'll get rid of it.